HeaderView.swift

```diff
@@ -35,5 +38,28 @@ class HeaderView: UIView {
             visualEffectView.frame = headerBackground.bounds
             headerBackground.addSubview(visualEffectView)
         }
+        
+        return headerBackground
+    }()
+    
+    // whenever a property that effects the header view changes, call this
+    //
+    private func updateHeaderBackground() {
+        headerBackground.removeFromSuperview()
+        headerBackground = nil
```

Comment:
setting this back to `nil` will trigger the lazy loading code again when adding as a subview, which will take into account the updated `blurStyle` and `image`